autoload/lsp/omni.vim

```diff
@@ -303,54 +303,64 @@ function! lsp#omni#get_vim_completion_items(options) abort
     let l:start_characters = [] " The mapping of item specific start_character.
     let l:vim_complete_items = []
     for l:completion_item in l:items
-        let l:expandable = get(l:completion_item, 'insertTextFormat', 1) == 2
-        let l:vim_complete_item = {
-            \ 'kind': get(l:kind_text_mappings, get(l:completion_item, 'kind', '') , ''),
-            \ 'dup': 1,
-            \ 'empty': 1,
-            \ 'icase': 1,
-            \ }
-        let l:range = lsp#utils#text_edit#get_range(get(l:completion_item, 'textEdit', {}))
```

Comment:
`lsp#utils#text_edit#get_range()` takes a lot of time. but it is used for only 'textEdit'. So move here.

https://github.com/prabirshrestha/vim-lsp/blob/c0482f0fb407c29a88875616175784a886f131c3/autoload/lsp/omni.vim#L312